### ISSUE TITLE
Core 3.1 No Schema on Index and MigrationHistory

### ISSUE THREAD

**Issue by stefannikolei** [oracle/dotnet-db-samples#117]
I have configured a DefaultSchema like this.

```c#
     protected override void OnModelCreating(ModelBuilder modelBuilder)
        {
            base.OnModelCreating(modelBuilder);

            // Configure default schema
            modelBuilder.HasDefaultSchema("MYSCHEMA");

            // Configure Table and Column Names
            foreach (var entityType in modelBuilder.Model.GetEntityTypes())
            {

                if (entityType.Name.ToUpper() != entityType.DisplayName().ToUpper())
                {
                    entityType.SetTableName(entityType.DisplayName().ToUpper());
                }


                foreach (var property in entityType.GetProperties())
                {
                    property.SetColumnName(property.GetColumnName().ToUpper());
                }
            }

            modelBuilder.ApplyConfigurationsFromAssembly(typeof(OracleDbContext).Assembly);
        }
```
Now when i create an Index i get following SQL:

`CREATE INDEX "IX_MYTABLE_CUSTOMERNO" ON "MYSCHEMA"."MYTABLE" ("CUSTOMERNO")`
shouldn't it be like this
`CREATE INDEX "MYSCHEMA"."IX_MYTABLE_CUSTOMERNO" ON "MYSCHEMA"."MYTABLE" ("CUSTOMERNO")`

The same problem is for the initial creation of the MigrationHistoryTable.

**Comment by alexkeh** [comment]
@stefannikolei 
Thanks for reporting the bug. I have filed Bug 32131871 to track the issue.

The migration history table has a different mechanism for specifying an alternative schema to be created in.
https://docs.microsoft.com/en-us/ef/core/managing-schemas/migrations/history-table

```
protected override void OnConfiguring(DbContextOptionsBuilder options)

    => options.UseOracle(

        connectionString,

        x => x.MigrationsHistoryTable("__MyMigrationsHistory", "mySchema"));
```

**Comment by alexkeh** [comment]
This bug has been fixed in the Oracle EF Core 5.21.1 release.
